The job **`x86_64-gnu-llvm-16`** failed! Check out the build log: [(web)](https://github.com/rust-lang/rust/actions/runs/7304780149/job/19907443302) [(plain)](https://github.com/rust-lang/rust/commit/6a0a0c2d79f1137e01d649485a415624567e29a1/checks/19907443302/logs)

<details><summary><i>Click to see the possible cause of the failure (guessed by this bot)</i></summary>

```plain
GITHUB_ACTION=__run_7
GITHUB_ACTIONS=true
GITHUB_ACTION_REF=
GITHUB_ACTION_REPOSITORY=
GITHUB_ACTOR=Urgau
GITHUB_API_URL=https://api.github.com
GITHUB_BASE_REF=master
GITHUB_ENV=/home/runner/work/_temp/_runner_file_commands/set_env_9d488d77-4ee0-4dd8-b10b-60da55f26b1a
GITHUB_EVENT_NAME=pull_request
GITHUB_EVENT_NAME=pull_request
GITHUB_EVENT_PATH=/home/runner/work/_temp/_github_workflow/event.json
GITHUB_GRAPHQL_URL=https://api.github.com/graphql
GITHUB_HEAD_REF=missing-feature-gate-sanitizer-cfi-cfgs
GITHUB_JOB=pr
GITHUB_PATH=/home/runner/work/_temp/_runner_file_commands/add_path_9d488d77-4ee0-4dd8-b10b-60da55f26b1a
GITHUB_REF=refs/pull/119235/merge
GITHUB_REF_NAME=119235/merge
GITHUB_REF_PROTECTED=false
---
GITHUB_SERVER_URL=https://github.com
GITHUB_SHA=bad42abd0deb9e2e6e71839a077f50f388664650
GITHUB_STATE=/home/runner/work/_temp/_runner_file_commands/save_state_9d488d77-4ee0-4dd8-b10b-60da55f26b1a
GITHUB_STEP_SUMMARY=/home/runner/work/_temp/_runner_file_commands/step_summary_9d488d77-4ee0-4dd8-b10b-60da55f26b1a
GITHUB_TRIGGERING_ACTOR=Urgau
GITHUB_WORKFLOW_REF=rust-lang/rust/.github/workflows/ci.yml@refs/pull/119235/merge
GITHUB_WORKFLOW_SHA=bad42abd0deb9e2e6e71839a077f50f388664650
GITHUB_WORKSPACE=/home/runner/work/rust/rust
GOROOT_1_19_X64=/opt/hostedtoolcache/go/1.19.13/x64
---
Built container sha256:9c3c93a371e5aed5c18185b24f130d95d5140dbd72a9b325e7b6b49e521a4faa
Looks like docker image is the same as before, not uploading
https://ci-caches.rust-lang.org/docker/7ebc15c01a233894034d277c8cce4e949f4e7791f66b4727c8fb6e058a0b8171d6152e1441d677cef0653843ceeee469c097b8699b2bb74249e674f6aa1a8813
sha256:9c3c93a371e5aed5c18185b24f130d95d5140dbd72a9b325e7b6b49e521a4faa
Setting extra environment values for docker:  --env ENABLE_GCC_CODEGEN=1 --env GCC_EXEC_PREFIX=/usr/lib/gcc/
[CI_JOB_NAME=x86_64-gnu-llvm-16]
##[group]Clock drift check
  local time: Fri Dec 22 23:57:00 UTC 2023
  network time: Fri, 22 Dec 2023 23:57:00 GMT
  network time: Fri, 22 Dec 2023 23:57:00 GMT
##[endgroup]
sccache: Starting the server...
##[group]Configure the build
configure: processing command line
configure: 
configure: build.configure-args := ['--build=x86_64-unknown-linux-gnu', '--llvm-root=/usr/lib/llvm-16', '--enable-llvm-link-shared', '--set', 'rust.thin-lto-import-instr-limit=10', '--enable-verbose-configure', '--enable-sccache', '--disable-manage-submodules', '--enable-locked-deps', '--enable-cargo-native-static', '--set', 'rust.codegen-units-std=1', '--set', 'dist.compression-profile=balanced', '--dist-compression-formats=xz', '--disable-dist-src', '--release-channel=nightly', '--enable-debug-assertions', '--enable-overflow-checks', '--enable-llvm-assertions', '--set', 'rust.verify-llvm-ir', '--set', 'rust.codegen-backends=llvm,cranelift,gcc', '--set', 'llvm.static-libstdcpp', '--enable-missing-tools', '--enable-new-symbol-mangling']
configure: target.x86_64-unknown-linux-gnu.llvm-config := /usr/lib/llvm-16/bin/llvm-config
configure: llvm.link-shared     := True
configure: rust.thin-lto-import-instr-limit := 10
configure: rust.codegen-units-std := 1
---
   Compiling addr2line v0.21.0
error: unexpected `cfg` condition name: `sanitizer_cfi_normalize_integers`
  --> library/std/src/sys/unix/thread_local_dtor.rs:32:11
   |
32 |     #[cfg(sanitizer_cfi_normalize_integers)]
   |
   |
   = help: expected names are: `backtrace_in_libstd`, `bootstrap`, `debug_assertions`, `doc`, `doctest`, `feature`, `miri`, `no_fp_fmt_parse`, `no_global_oom_handling`, `no_rc`, `no_sync`, `overflow_checks`, `panic`, `proc_macro`, `relocation_model`, `sanitize`, `stdarch_intel_sde`, `target_abi`, `target_arch`, `target_endian`, `target_env`, `target_family`, `target_feature`, `target_has_atomic`, `target_has_atomic_equal_alignment`, `target_has_atomic_load_store`, `target_os`, `target_pointer_width`, `target_thread_local`, `target_vendor`, `test`, `unix`, `windows`
   = note: `-D unexpected-cfgs` implied by `-D warnings`
   = help: to override `-D warnings` add `#[allow(unexpected_cfgs)]`
error: unexpected `cfg` condition name: `sanitizer_cfi_normalize_integers`
  --> library/std/src/sys/unix/thread_local_dtor.rs:34:15
   |
   |
34 |     #[cfg(not(sanitizer_cfi_normalize_integers))]

error[E0635]: unknown feature `cfg_sanitizer_cfi`
   --> library/std/src/lib.rs:273:12
    |
    |
273 | #![feature(cfg_sanitizer_cfi)]

For more information about this error, try `rustc --explain E0635`.
error: could not compile `std` (lib) due to 3 previous errors
Build completed unsuccessfully in 0:00:15
```

</details>

--- a/library/std/src/lib.rs
+++ b/library/std/src/lib.rs
@@ -265,6 +265,7 @@
 //
 // Language features:
 // tidy-alphabetical-start
+#![cfg_attr(not(bootstrap), feature(cfg_sanitizer_cfi))]
 #![feature(alloc_error_handler)]
 #![feature(allocator_internals)]
 #![feature(allow_internal_unsafe)]

--- a/library/std/src/sys/unix/thread_local_dtor.rs
+++ b/library/std/src/sys/unix/thread_local_dtor.rs
@@ -11,7 +11,7 @@
 // Note, however, that we run on lots older linuxes, as well as cross
 // compiling from a newer linux to an older linux, so we also have a
 // fallback implementation to use as well.
-#[allow(unexpected_cfgs)]
+#[cfg_attr(bootstrap, allow(unexpected_cfgs))]
 #[cfg(any(
     target_os = "linux",
     target_os = "android",

